src/main/frontend/routes.cljs

```diff
@@ -78,6 +79,14 @@
    ["/import"
     {:name :import
      :view setups/importer}]
+   
+   ["/bug-report"
+    {:name :bug-report
+     :view bug-report/bug-report}]
+   
+    ["/bug-report-tool/:tool"
```

Comment:
**non-blocking**: Better hide it under "Developer mode".

Reply:
IMO hiding it in Dev mode is not a good idea 👀

Reply:
IMO this is a feature orienting to the users. 🤔